packages/core/src/__tests__/document.test.ts

```diff
@@ -488,7 +489,7 @@ describe('Document', () => {
       })
     })
 
-    it("Neither log is anchored, no nonces", async () => {
```

Comment:
instead of `neither log is anchored, no nonces`, this should be `neither log is anchored, logs of the same length`

Reply:
reflected in the PR [657](https://github.com/ceramicnetwork/js-ceramic/pull/657)